server-plugins/contact-resources/src/index.ts

```diff
@@ -275,6 +277,54 @@ export async function OnEmployeeUpdate (tx: Tx, control: AsyncTriggerControl): P
   return result
 }
 
+/**
+ * @public
+ */
+export async function OnChannelUpdate (tx: Tx, control: TriggerControl): Promise<Tx[]> {
+  if (tx._class !== core.class.TxUpdateDoc) {
+    return []
+  }
+
+  const uTx = tx as TxUpdateDoc<Channel>
+
+  if (!control.hierarchy.isDerived(uTx.objectClass, contact.class.Channel)) {
+    return []
+  }
+
+  const result: Tx[] = []
+
+  if (uTx.operations.$inc?.items !== undefined) {
+    const doc = (await control.findAll(uTx.objectClass, { _id: uTx.objectId }, { limit: 1 }))[0]
```

Comment:
Should we have some standalone logic in notification server plugin? and provide this information as Mixin for Classes?